### ISSUE TITLE
Public keys of type RSAES-OAEP is currently not supported

### ISSUE THREAD

**Issue by space88man** [randombit/botan#1943]
There is a sample OAEP certificate in https://github.com/openssl/openssl/issues/7907#issuecomment-450211689

I was trying to use cli to view it:

```
# botan is 2.9.0 (Fedora 30)
botan cert_info sample.crt 

X509_Certificate parsing failed CERTIFICATE decoding failed with Decoding subject public keys of type RSAES-OAEP is currently not supported
```

**Comment by randombit** [comment]
Format described in https://tools.ietf.org/html/rfc4055#section-4.1

**Comment by randombit** [comment]
I think the check that rejects decoding is just wrong. We would still be missing support for decoding the mandated OAEP params but the cert should still be parsed.

**Comment by space88man** [comment]
This is great - thanks for the quick turnaround. In any case decoding the parameters should not be an issue  at least for certificate's subjectPublicKeyInfo field as that must not include any parameters. (https://tools.ietf.org/html/rfc5756 Section 3).

**Comment by randombit** [comment]
Oh perfect then we don't even need to do anything further. Always a happy result. Thanks for mentioning the updated spec.

**Comment by randombit** [comment]
Fixed now in master with merge of #1944 thanks for reporting this.
